src/hera/workflow.py

```diff
@@ -71,6 +72,7 @@ def __init__(
         security_context: Optional[WorkflowSecurityContext] = None,
         image_pull_secrets: Optional[List[str]] = None,
         workflow_template_ref: Optional[str] = None,
+        ttl_strategy: Optional[Dict[str, int]] = None,
```

Comment:
What do you think about making a custom Hera object representation of the TTL strategy? I think that will allow Hera to construct the TTL strategy using something like a `get_argo_spec`, similar to [Env](https://github.com/argoproj-labs/hera-workflows/blob/main/src/hera/env.py), which is a nice abstraction that supports creating custom logic that is aimed at helping the user have an easier time creating workflows!